app/models/server.rb

```diff
@@ -27,4 +32,17 @@ class Server < ApplicationRecord
     with: /([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})/,
     message: 'is not a valid MAC address'
   }
+
+  after_initialize :assign_defaults
+
+  # There is actually only ever one responsible user
+  # This method is a convenience accessor to allow for easier array concatination
+  # It is also recommended to use this method instead of :responsible because we might want to allow multiple responsible users in the future.
+  def responsible_users
+    responsible.nil? ? [] : [responsible]
+  end
+
+  def assign_defaults
```

Comment:
Can we assign the default using ActiveRecord? E.g. disallow `null` values in the db column, set '' as the default?

Reply:
Add a comment on why this is needed and can't be done via ActiveRecord!